presqt/targets.json

```diff
@@ -1,12 +1,6 @@
 [
   {
     "name": "osf",
-    "read": true,
-    "write": true
-  },
-  {
-    "name": "curate_nd",
-    "read": true,
-    "write": false
+    "resource_collection": true
```

Comment:
Maybe it would be good to put the "supported actions" in a nested object like so...

{
    "name": "osf",
    "supportedActions": {
        resource_collection": true
    }
}

This would just keep the top level of the documented cleaner if we wanted to add other non-functionality related metadata in the future.

Just a thought.